docker/ubuntu/18.04/entrypoint.sh

```
@@ -28,10 +28,37 @@ fi
 
 [ "${USER_NAME:-root}" != "root" ] || die "Invalid \$USER_NAME (${USER_NAME})"
 
+# helper to find existing group by GID
+find_group_by_gid() {
+	local name
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)